docs/install/kata-containers-3.0-rust-runtime-installation-guide.md

```diff
@@ -83,6 +83,16 @@ $ make && sudo make install
 ```
 After running the command above, the default config file `configuration.toml` will be installed under `/usr/share/defaults/kata-containers/`,  the binary file `containerd-shim-kata-v2` will be installed under `/usr/local/bin/` .
 
+### Build Shim Without Builtin Dragonball VMM
+
+By default, `runtime-rs` includes the `Dragonball` VMM. However, when a `non-dragonball` hypervisor is set as the default hypervisor, `dragonball` will not be compiled. For example, if you want to use `qemu` as the default hypervisor,  use `make DEFAULT_HYPERVISOR=qemu`. 
+
+```bash
+$ cd kata-containers/src/runtime-rs
+$ make DEFAULT_HYPERVISOR=qemu
```

Comment:
`DEFAULT_HYPERVISOR` looks a little bit weird for me. What about `HYPERVISOR`?

Reply:
Currently, there is no variable named HYPERVISOR in the Makefile, and DEFAULT_HYPERVISOR controls the default hypervisor configuration. It appears multiple times in the Makefile. If we introduce HYPERVISOR, does it mean we need to replace all instances of DEFAULT_HYPERVISOR with HYPERVISOR? Otherwise, there will be two variables controlling the hypervisor in use.

Reply:
Yes. I think we have to do that.

Reply:
The modifications have already been made.